core/notification/mocks/idempotency_repository.go

```diff
@@ -0,0 +1,163 @@
+// Code generated by mockery v2.14.0. DO NOT EDIT.
+
+package mocks
```

Comment:
why not have just _mock.go in same pkg ?

Reply:
I think it is better to place it in mocks only to keep generated code out of the core package.